lib/dest/writeContents/index.js

```diff
@@ -60,17 +60,16 @@ function writeContents(writePath, file, cb) {
   }
 
   function isErrorFatal(err) {
+    var isFatal = true;
```

Comment:
I changed it at https://github.com/gulpjs/vinyl-fs/commit/00ece7ef6b794fd1999695fde2d88bba8d69f5d0